bundles/org.openhab.binding.plex/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -65,6 +68,25 @@
 		<description>The title of media that is playing</description>
 		<state readOnly="true"/>
 	</channel-type>
+	<channel-type id="ratingKey" advanced="true">
+		<item-type>String</item-type>
+		<label>Rating Key</label>
+		<description>The unique key in the Plex library identifying the media that is playing</description>
+		<state readOnly="true"/>
+	</channel-type>
+	<channel-type id="parentRatingKey" advanced="true">
+		<item-type>String</item-type>
+		<label>Parent Rating Key</label>
+		<description>The unique key in the Plex library identifying the parent (TV show season or album) of the media that is
+			playing</description>
+		<state readOnly="true"/>
+	</channel-type>
+	<channel-type id="grandparentRatingKey" advanced="true">
+		<item-type>String</item-type>
+		<label>Grandparent Rating Key</label>
+		<description>The unique key in the Plex library identifying the grandparent (TV show) of the media that is playing</description>
+		<state readOnly="true"/>
+	</channel-type>
```

Comment:
Upgrade instructions are needed to automatically add the new channels. See here for an example: https://github.com/openhab/openhab-addons/pull/14799/files#diff-f5712db66efe98f83c01f11166123dcdf146398d811ef519f2b2836a39f59fe3

Reply:
Done. And I tested both upgrading from a thing created before this change, and a thing created after this change.